Makefile

```diff
@@ -0,0 +1,86 @@
+.DEFAULT_GOAL := spec
+
+
+# builds for the current platform
+build:
+	go install
+
+# makes a new binary release
```

Comment:
How about `builds the artifacts for a new release`. It doesn't actually `make` a release by publishing or anything.

Reply:
done